code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -104,6 +104,12 @@ God bless America.
 		else if(!frying && user.transferItemToLoc(I, src))
 			to_chat(user, "<span class='notice'>You put [I] into [src].</span>")
 			frying = new/obj/item/reagent_containers/food/snacks/deepfryholder(src, I)
+			//setup food quality for item depending on if it's edible or not
+			if(isfood(I))
+				var/obj/item/reagent_containers/food/original_food = I
+				frying.adjust_food_quality(original_food.food_quality) //food quality remains unchanged until degree of frying is calculated
+			else
+				frying.food_quality = 10 //inedible fried item has low quality
```

Comment:
er is this supposed to not use adjust_food_quality for some reason because you're using that everywhere else

Reply:
all changes to it should be done by adjust food quality ill change that